cstrml/tee/src/lib.rs

```diff
@@ -168,7 +168,7 @@ decl_module! {
         }
 
         #[weight = 1_000_000]
-        fn report_works(origin, work_report: WorkReport) -> DispatchResult {
+        pub fn report_works(origin, work_report: WorkReport) -> DispatchResult {
```

Comment:
Only for test?

Reply:
yes. If we want to test payment, we have to use tee and market to pass the checks. I tried to remove `pub` and looks like we cannot do that. `register_identity` has pub too.